src/components/Map/Map.js

```diff
@@ -67,23 +74,34 @@ const Map = ({ permalinkParam }) => {
           dispatch(setZoom(evt.map.getView().getZoom()));
         }}
       />
-      <TextField
-        label="Tiles url"
-        className={classes.input}
-        variant="outlined"
-        value={tilesUrl}
-        onChange={(evt) => {
-          setTilesUrl(evt.target.value);
-        }}
-      />
+      <div className={classes.inputWrapper}>
+        <FormLabel focused={focused} className={classes.label}>
+          GL Style URL
+        </FormLabel>
+        <TextField
+          className={classes.input}
+          onFocus={() => onFocused(true)}
+          onBlur={() => onFocused(false)}
+          placeholder="Paste the URL to a GL Style JSON here"
+          variant="outlined"
+          value={tilesUrl}
+          onChange={(evt) => {
+            setTilesUrl(evt.target.value);
+          }}
+        />
+      </div>
```

Comment:
I dont'think you need the form label
```
<TextField
          id="outlined-full-width"
          label="Label"
          style={{ margin: 8 }}
          placeholder="Placeholder"
          fullWidth
          margin="normal"
          InputLabelProps={{
            shrink: true,
          }}
          variant="outlined"
        />
```

Reply:
if you pass the label, it will overlap with the placeholder.
This way we are allowed to have both

Reply:
Did you test what I send you ?

Reply:
Yes I changed the code with the inspiration opf your example thanks :+1: